src/index.html

```diff
@@ -10,13 +10,103 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+          class="logo--image"
+        />
+      </a>
+      <nav class="menu">
+        <li class="menu--li">
+          <a
+            href="#"
+            class="menu-link is-active"
+          >
+            Apple
+          </a>
+        </li>
+        <li class="menu--li">
+          <a
+            href="#"
+            class="menu-link"
```

Comment:
Fix classes everywhere
```suggestion
        <li class="nav__item">
          <a
            href="#"
            class="nav__link"
```